components/ai_chat/core/common/mojom/ai_chat.mojom

```diff
@@ -105,6 +105,13 @@ struct Conversation {
   bool has_content;
   // Model key, if different than default
   string? model_key;
+  // Indicates the total number of tokens for the conversation.
+  // This info is determined by the server side and applies to the whole conversation.
+  uint64 total_tokens;
+  // Indicates the number of tokens that were trimmed from the conversation.
+  // This info is determined by the server side and applies to the whole conversation.
+  // Things that can be trimmed include associated content(s) and server side things like search results.
+  uint64 trimmed_tokens;
```

Comment:
Note that I had originally coded this inside AssociatedContents but since had to change it to apply to the whole conversation. The reason is that this info is global to the whole conversation and not only related to a single associated content.  Also the server considers associated content for things like search results too, and the client doesn't consider that associated content. So this info can come even when there is no client side associated content.